yarn-project/end-to-end/src/e2e_cross_chain_messaging.test.ts

```diff
@@ -233,18 +234,6 @@ describe('e2e_cross_chain_messaging', () => {
     // Wait for the message to be available for consumption
     await crossChainTestHarness.makeMessageConsumable(msgHash);
 
-    const content = sha256ToField([
```

Comment:
It's a pity to remove this, but it's unavoidable until we can assert with format-strings. Right now we get this message even in public because public is doing a "get merkle proof" just like in private in `process_l1_to_l2_message`, so the oracle does the string formatting for the error. In the AvmContext, we don't do this. We err with a normal `assert`.